graphql-server/src/schemas/queries/patients.ts

```diff
@@ -177,3 +186,88 @@ export async function queryDashboardPatients({
     await session.close();
   }
 }
+
+export async function queryPatientIdsTriplets(searchVals: Array<string>) {
+  const searchValsWithoutCDash = searchVals.map((searchVal) =>
+    searchVal.startsWith("C-") ? searchVal.slice(2) : searchVal
+  );
+  const searchValList = searchValsWithoutCDash
+    .map((searchVal) => `'${searchVal}'`)
+    .join(",");
+  const query = `
+    SELECT
+      CMO_PATIENT_ID,
+      DMP_PATIENT_ID,
+      MRN
+    FROM
+      ${props.databricks_phi_id_mapping_table}
+    WHERE
+      DMP_PATIENT_ID IN (${searchValList})
+      OR MRN IN (${searchValList})
+      OR CMO_PATIENT_ID IN (${searchValList})
+  `;
+  const patientIdsTriplets = await queryDatabricks<PatientIdsTriplet>(query);
+  patientIdsTriplets.forEach((patientIdTriplet) => {
+    patientIdTriplet.CMO_PATIENT_ID = `C-${patientIdTriplet.CMO_PATIENT_ID}`;
+  });
+  return patientIdsTriplets;
+}
+
+export async function queryAnchorSeqDatesByDmpPatientId(
+  dmpPatientIds: Array<string>
+) {
+  const dmpPatientIdsList = dmpPatientIds
+    .map((dmpPatientId) => `'${dmpPatientId}'`)
+    .join(",");
+  const query = `
+    SELECT
+      DMP_PATIENT_ID,
+      ANCHOR_SEQUENCING_DATE
+    FROM
```

Comment:
PMs want to be able to search by MRN and DMP sample ID as well. following up with @n1zea144 if we need a new view for this or if we can alter the current view we're using to include this info as well

Reply:
resolving this comment as this work is in progress